plate_mapper/manual.html

```diff
@@ -236,6 +236,46 @@
     <li>The content of this file can be directly copied and pasted into the final mapping file template (Excel spreadsheet format).</li>
   </ul>
 
+  <h3>(optional) <b><u>names</u></b>: sample name list</h3>
+  <div class="table" style="height:240px">
+    <table>
+      <tr><td class="row-head"></td><td class="col-head">A</td><td class="col-head">B</td><td class="col-head">C</td><td class="col-head">D</td><td class="col-head">E</td><td class="col-head">F</td><td class="col-head">G</td></tr>
+      <tr><td class="row-head">1</td><td>1.POOP</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">2</td><td>1.E1</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">3</td><td>1.ST1</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">4</td><td>1.D6</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">5</td><td>1.I2</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">6</td><td>1.CE4</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">7</td><td>1.CO6</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">8</td><td>1.V2</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">9</td><td>1.CE4</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">...</td><td colspan="7">...</td></tr>
+      <tr><td class="row-head">87</td><td>4.BR1</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">88</td><td>4.BR2</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+      <tr><td class="row-head">89</td><td>4.BR3</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+    </table>
+  </div>
+  <ul>
+    <li>This function is to validate whether the samples in the mapping file are identical to this given sample name list. Specifically, this function will report sample names that 1) are present in the mapping file but not in the name list, 2) are present in the name list but not in the mapping file, 3) have duplicates in the mapping file.</li>
```

Comment:
Duplicates in the sample name list will be ignored, as I used a Python set to parse these names.